### ISSUE TITLE
Backup not working correctly.

### ISSUE THREAD

**Issue by n8udd** [TablePlus/TablePlus#2565]
Table plus doesn't appear to be backing up the data as expected.

### Please answer the questions below, it helps us to keep track of the issue. Please do not ignore it or your issue will be closed.

1. Which driver are you using and version of it (Ex: PostgreSQL 10.0):
MySQL 8.0.23

2. Which TablePlus build number are you using (the number on the welcome screen, Ex: build 81):
396

3. The steps to reproduce this issue:
As in the docs [here](https://docs.tableplus.com/gui-tools/backup-and-restore), I'm trying to backup the db.
![image](https://user-images.githubusercontent.com/2293192/144871685-bb83eb85-4659-4c36-90a7-9a5965f9a29a.png)

I've tried with version 5.7 and 8.0, but every database that I backup saves a `.dump` file that is 203 bytes in size, and contains:

```
Usage: mysqldump [OPTIONS] database [tables]
OR     mysqldump [OPTIONS] --databases [OPTIONS] DB1 [DB2 DB3...]
OR     mysqldump [OPTIONS] --all-databases [OPTIONS]
For more options, use mysqldump --help
```
It makes out that the data is being backed up in the gui:
![image](https://user-images.githubusercontent.com/2293192/144871910-90bf1109-5e1e-41e6-9dcd-57ae86a39609.png)

Noted: If the bug is related to data, please attach an example SQL data.

**Comment by peterthomson** [comment]
Yep. Also having this issue. Maybe something to do with a local database that doesn't have a password. Are you running the backup on a local database?

**Comment by huyphams** [comment]
Hi @n8udd could you please remove the --all-databases flag and try again please?

**Comment by peterthomson** [comment]
Ironically, the only way I could work around the issue to get a dump that wasn't 'blank' was to add the All Databases flag :)

**Comment by nathan-mill** [comment]
Seeing the exact same issue attempting to backup a mySQL RDS instance with or without the --all-databases flag. The dump that TablePlus saves out is microscopic and contains nothing of any value.
